tsconfig.json

```diff
@@ -2,15 +2,16 @@
   "compileOnSave": false,
   "compilerOptions": {
     "baseUrl": "./",
+    "downlevelIteration": true,
```

Comment:
You coded this rule yourself or is it enabled by default now?

Reply:
I think that was automatic, by upgrading angular to 8.